### PR TITLE
ASE: Moving some global variables declaration out of ase_common.h 

### PR DESCRIPTION
**Pull request by lzhan55**
ASE: Moving some global variables declaration out of ase_common.h with some other changes.

* This changes has been validated on external GitHub with help_fpga test
* Will check it on Jenkin soon

### CODE REVIEW

**Comment by lzhan55** [comment]
Jenkins' ase-testing tests passed for this pull requests.

File: ase/sw/protocol_backend.c [hunk 1]
```diff
@@ -40,6 +40,18 @@
...
+int app2sim_mmioreq_rx;		// MMIO Request path
+int sim2app_mmiorsp_tx;		// MMIO Response path
+int app2sim_umsg_rx;		// UMSG    message queue in RX mode
+int app2sim_portctrl_req_rx;	// Port Control messages in Rx mode
+int app2sim_dealloc_rx;
+int sim2app_dealloc_tx;
+int sim2app_portctrl_rsp_tx;
+int sim2app_intr_request_tx;
+int intr_event_fds[MAX_USR_INTRS];
+
```
Comment (michael-adler) [inline ase/sw/protocol_backend.c]: Should most of these non-globals be static?

Comment (lzhan55) [reply ase/sw/protocol_backend.c]: Yes, I can change these variables to be static if it is only used in one module.

**Comment by deepakcu84** [comment]
It's okay to merge this one. In the future, can we also have a link to PSG regtests in the review comments?

**Comment by lzhan55** [comment]
Yes, for the next pull requests with Windows support, I will run full regression tests on MCP and DCP

**Pull request merged**